packages/@atjson/offset-annotations/src/utils/aspect-ratios.ts

```diff
@@ -0,0 +1,85 @@
+const ASPECT_RATIOS = [
```

Comment:
Can you make a comment that it is important that this list remain sorted?

Reply:
I'm going to make it so sorting doesn't matter